internal/plugin/plugin.go

```diff
@@ -321,16 +322,33 @@ func parsePluginsEnvVar(pluginsEnv string) ([]includePlugin, error) {
 	return includes, nil
 }
 
+func getLatestPluginVersionsByName(plugins []*Plugin) map[string]string {
```

Comment:
Nit, wouldn't mind a comment with an example of the final map.

Reply:
Added.